schematic/models/validate_attribute.py

```diff
@@ -1589,11 +1716,33 @@ def _run_validation_across_target_manifests(
             target_column, pd.core.series.Series: Empty target_column to fill out in this function
         Returns:
             start_time, float: start time in fractional seconds
-            validation_store, Union[
+            validation_output, Union[
                 tuple[dict[str, pd.core.series.Series], list[str], dict[str, pd.core.series.Series]],
                 tuple[dict[str, pd.core.series.Series], dict[str, pd.core.series.Series],
                     dict[str, pd.core.series.Series]]:
                     validation outputs, exact types depend on scope,
+            if not target_attribute_in_manifest:
+                return (start_time, target_attribute_in_manifest)
+            else:
+                # Store outputs according to the scope for which they are used.
+                if "set" in rule_scope:
+                    validation_store = (
+                        missing_manifest_log,
+                        present_manifest_log,
+                        repeat_manifest_log,
+                    )
+
+                elif "value" in rule_scope:
+                    # From the concatenated target column, for value scope, run validation
+                    (
+                        missing_values,
+                        duplicated_values,
+                        repeat_values,
+                    ) = self._run_validation_across_targets_value(
+                        manifest_col=manifest_col,
+                        concatenated_target_column=target_column,
+                    )
+                    validation_store = (missing_values, duplicated_values, repeat_values)
```

Comment:
Is this code that got commented accidentally?

Reply:
No, I put it here while I was moving things around and then forgot, its gone now.